Makefile

```diff
@@ -0,0 +1,151 @@
+# ====================================================================
+# Copyright oVirt Authors
+# SPDX-License-Identifier: Apache-2.0
+# ====================================================================
+
+#
+# CUSTOMIZATION-BEGIN
+#
+# Keycloak version specification
+KEYCLOAK_VERSION="15.0.2"
+
+# RPM version specification
+RPM_VERSION="${KEYCLOAK_VERSION}"
+RPM_RELEASE="1"
+
+EXTRA_BUILD_FLAGS=
+BUILD_VALIDATION=1
+
+PACKAGE_NAME=ovirt-engine-keycloak
+
+PYTHON=$(shell which python3 2> /dev/null)
+PREFIX=/usr/local
+SYSCONF_DIR=$(PREFIX)/etc
+DATAROOT_DIR=$(PREFIX)/share
+PKG_DATA_DIR=$(DATAROOT_DIR)/ovirt-engine-keycloak
+KEYCLOAK_OVERLAY_ZIP="keycloak-overlay-$(KEYCLOAK_VERSION).zip"
+KEYCLOAK_OVERLAY_URL="https://github.com/keycloak/keycloak/releases/download/${KEYCLOAK_VERSION}/${KEYCLOAK_OVERLAY_ZIP}"
+#
+# CUSTOMIZATION-END
+#
+BUILD_FLAGS:=$(BUILD_FLAGS) $(EXTRA_BUILD_FLAGS)
+
+TARBALL=$(PACKAGE_NAME)-$(RPM_VERSION).tar.gz
+BUILD_FILE=tmp.built
+
+
+.SUFFIXES:
+.SUFFIXES: .in
+
+.in:
+	sed \
+	-e "s|@KEYCLOAK_VERSION@|$(KEYCLOAK_VERSION)|g" \
+	-e "s|@KEYCLOAK_OVERLAY_ZIP@|$(KEYCLOAK_OVERLAY_ZIP)|g" \
+	-e "s|@DATAROOT_DIR@|$(DATAROOT_DIR)|g" \
+	-e "s|@PKG_DATA_DIR@|$(PKG_DATA_DIR)|g" \
+	-e "s|@PKG_STATE_DIR@|$(PKG_STATE_DIR)|g" \
+	-e "s|@RPM_VERSION@|$(RPM_VERSION)|g" \
+	-e "s|@RPM_RELEASE@|$(RPM_RELEASE)|g" \
+	-e "s|@PACKAGE_NAME@|$(PACKAGE_NAME)|g" \
+	-e "s|@KEYCLOAK_VERSION@|$(KEYCLOAK_VERSION)|g" \
+	-e "s|@KEYCLOAK_OVERLAY_PKG@|$(KEYCLOAK_OVERLAY_PKG)|g" \
```

Comment:
Why do you have KEYCLOAK* replacements defined twice?

Reply:
mistake -> fixed